pallets/parachain-staking/src/lib.rs

```diff
@@ -2532,27 +2539,53 @@ pub mod pallet {
                 as Box<dyn VotingSessionManager<T::AccountId, T::BlockNumber>>
         }
 
-        pub fn try_get_growth_data(growth_id: &GrowthId) -> Result<GrowthData<T>, Error<T>> {
-            if <Growth<T>>::contains_key(growth_id.period) {
-                let growth_info = <Growth<T>>::get(growth_id.period);
-                if growth_info.number_of_accumulations < 0u32 {
-                    Err(Error::<T>::AccumulationIsZero)?
-                }
+        pub fn trigger_growth_on_t1(growth_period: &u32) -> Result<(), Error<T>> {
+            if <Growth<T>>::contains_key(growth_period) {
+                let growth_info = <Growth<T>>::get(growth_period);
+                               
+                let rewards_in_period_128 = TryInto::<u128>::try_into(growth_info.total_staker_reward)
+                .map_err(|_| Error::<T>::ErrorConvertingBalance)?;
 
-                let average_staked =  growth_info.total_stake_accumulated / growth_info.number_of_accumulations.into();
+                let average_staked_in_period_128 = TryInto::<u128>::try_into(growth_info.total_stake_accumulated / growth_info.number_of_accumulations.into())
+                .map_err(|_| Error::<T>::ErrorConvertingBalance)?;
+                
+                let trigger_growth = EthTransactionType::TriggerGrowth(TriggerGrowthData::new(
+                    rewards_in_period_128,
+                    average_staked_in_period_128,
+                    *growth_period
+                ));
+
+                let tx_id =  Some(T::CandidateTransactionSubmitter::reserve_transaction_id(&trigger_growth).map_err(|_| Error::<T>::ErrorReservingId)?);                            
                 let added_by = 
                     AVN::<T>::calculate_primary_validator(<frame_system::Pallet<T>>::block_number())
                     .map_err(|_| Error::<T>::ErrorCalculatingPrimaryValidator)?;
+                
+                let quorum = calculate_two_third_quorum(AVN::<T>::validators().len() as u32);
```

Comment:
We have introduced and merged PRs to reduce the quorum to 1/3. Should this not reflect that?

Reply:
@alexpArtos - those are on [a feature branch](https://github.com/Aventus-Network-Services/avn-parachain/pull/225), so the growth work will use legacy methods for now. Then we can convert it along with the other author functions later.